### PR TITLE
Feature State Profile Awards Over Time Section

### PR DESCRIPTION
**Pull request by dtrinh50**
https://federal-spending-transparency.atlassian.net/browse/DEV-943

- Implemented Awards Over Time section

- [x] Code Review

- [x] Design Review

### CODE REVIEW

File: src/js/components/state/spendingovertime/StateTimeVisualizationSection.jsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+            // width changed, update the visualization width
+            this.setState({
+                windowWidth,
+                visualizationWidth: this.sectionHr.offsetWidth
+            });
+        }
+    }
+
+    render() {
+        return (
```
Comment (kevinli-work) [inline src/js/components/state/spendingovertime/StateTimeVisualizationSection.jsx]: Please refactor this component to follow BEM CSS class naming.

File: src/js/helpers/fiscalYearHelper.js [hunk 2]
```diff
@@ -114,3 +114,8 @@ export const convertDateToQuarter = (date) => {
 
     return quarter;
 };
+
+export const getTrailingTwelveMonths = () => {
```
Comment (kevinli-work) [inline src/js/helpers/fiscalYearHelper.js]: Please include a unit test for this.

File: tests/containers/state/StateTimeVisualizationSectionContainer-test.jsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+
+import { mockActions, mockRedux, mockYears, mockQuarters, mockMonths } from './mockData';
+
+jest.mock('helpers/searchHelper', () => require('./mockSearchHelper'));
+
+// mock the child component by replacing it with a function that returns a null element
+jest.mock('components/state/spendingovertime/StateTimeVisualizationSection', () => jest.fn(() => null));
+
+
+jest.mock('components/search/visualizations/time/TimeVisualizationSection', () =>
```
Comment (kevinli-work) [inline tests/containers/state/StateTimeVisualizationSectionContainer-test.jsx]: Why does this file need to be mocked?

File: tests/containers/state/StateTimeVisualizationSectionContainer-test.jsx [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+        });
+
+        container.setProps(nextProps);
+
+        await container.instance().apiRequest.promise;
+
+        expect(fetchData).toHaveBeenCalledTimes(1);
+    });
+
+    describe('parseData', () => {
```
Comment (kevinli-work) [inline tests/containers/state/StateTimeVisualizationSectionContainer-test.jsx]: Can you write additional, specific unit tests for the `generateXXX` functions you're using inside `parseData` that are implemented by `StateTimeVisualizationSectionContainer`?

**Comment by kevinli-work** [comment]
Also, review https://en.bem.info/methodology/quick-start/

**Comment by gitapablabah** [comment]
Awards Over Time section approved for design. TY.

**Pull request merged**